waffle/tests/test_templates.py

```diff
@@ -48,19 +48,7 @@ def test_no_request_context(self):
         assert 'switch off' in content
         assert 'sample' in content
 
-    @override_settings(TEMPLATE_LOADERS=(
-        'jingo.Loader',
-        'django.template.loaders.filesystem.Loader',
-        'django.template.loaders.app_directories.Loader',
-    ))
-    @mock.patch.object(template.loader, 'template_source_loaders', None)
-    def test_jingo_tags(self):
-        """
-            We're manually changing default TEMPLATE_LOADERS to enable jingo
-
-            template_source_loaders needs to be patched to None, otherwise
-            TEMPLATE_LOADERS won't be overriden.
-        """
```

Comment:
The test did fail with django 1.8 until I added the django-jinja bits though. So I am pretty sure the test is good.